src/Services/ServiceListParameters/ServiceListParameter.php

```diff
@@ -65,7 +65,7 @@ protected function appendParameter(?int $groupKey, ?string $groupType, string $n
     {
         if (! is_null($value))
         {
-            if (is_callable($value))
+            if (method_exists($this, $value))
```

Comment:
we were checking if `$value` is a function, why is it like this now?

even if it returns `true`, further down you are executing `$value` itself, and not `$this->{$value}`

Reply:
Since, we couldn't find any case where this will be used, and there's nothing regarding this on docs I've removed the check. Tested with most of the payments and everything seem to work fine.

Reply:
In the SDK, it seems that it has not been used, but this might have been used by a user, as I don't believe it was there without a reason